.github/workflows/all_plugins.yaml

```diff
@@ -0,0 +1,60 @@
+name: all_plugins
```

Comment:
I think now would be a good time to add the GitHub actions CI now after fixing all the analyzer error/violations to make sure that future PR does not add any further violations/error which decreases the quality of our codebase!